blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -9,7 +9,7 @@
 
 #pragma once
 
-#include <map>
+#include <optional>
```

Comment:
include here required to fix Windows builds. My fault. I broke this in an earlier PR as I hadn't checked Windows for a while. Used to CI telling me when I've been stupid. Apologies, I shall be more careful in future and ensure I test thoroughly on all 3 platforms!

Reply:
Heh, all good, sounds like it's really about time for me to handle #22 :)

Reply:
Ha yeah what a pain. I've done a fair amount of Jenkins in my time so shout if I can help with anything. No idea what you're thinking of using